package.json

```diff
@@ -102,6 +102,21 @@
         "description": "Displays test coverage in the current package."
       },
       {
+        "command": "go.test.generate.package",
+        "title": "Go: Generates unit tests (package)",
```

Comment:
Having parenthesis/brackets breaks autocomplete while typing commands in the command pallete. 
#495 fixed this for the test commands
Can you update the title for your commands as well?
example: Go: Generates unit tests for package/file/function